packages/react-flicking/src/react-flicking/CrossGroup.tsx

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2015 NAVER Corp.
+ * egjs projects are licensed under the MIT license
+ */
+import React, { ReactNode } from "react";
+
+const CrossGroup = React.forwardRef((props: { children?: ReactNode }, ref: React.Ref<HTMLDivElement>) => <div ref={ref}>{props.children}</div>);
```

Comment:
It seems that CrossGroup should also have Element properties (className).

Reply:
Fixed to use properties of elements in CrossGroup.